### PR TITLE
Simplify simplified tooling - remove return of TDR when encrypting

### PR DESCRIPTION
**Pull request by derekpierre**
**Type of PR:**
- [ ] Bugfix
- [ ] Feature
- [ ] Documentation
- [X] Other

**Required reviews:** 
- [ ] 1
- [X] 2
- [ ] 3

**What this does:**
Removes the ThresholdDecryptionRequest being returned by Enrico.

**Issues fixed/closed:**
Based over #3174 .

Related to #3143 

**Why it's needed:**
Threshold Decryption Requests should not be returned by Enrico. It is really a decryption object that Bob populates because it contains `variant` and the `context` dictionary (think Bob signing to prove ownership of a wallet). Therefore, the TDR should only be constructed by Bob and not Enrico.

**Notes for reviewers:**
> What should reviewers focus on? 
> Is there a particular commit/function/section of your PR that requires more attention from reviewers?

### CODE REVIEW

File: nucypher/characters/chaotic.py [hunk 1]
```diff
@@ -174,55 +167,54 @@ def gather_encrypted_decryption_shares(
...
+        # We can obtain the transcripts from the side-channel (deserialize) and aggregate them
+        validator_messages = [
+            ferveo.ValidatorMessage(validator, transcript)
+            for validator, transcript in self._learner._dkg_insight.transcripts
+        ]
+        aggregate = ferveo.AggregatedTranscript(validator_messages)
+        assert aggregate.verify(
+            self._learner._dkg_insight.shares_num,
+            # TODO this list should have to be passed again (either make `verify` static or use list
+            #  provided in constructor
```
Comment (piotr-roslaniec) [inline nucypher/characters/chaotic.py:186]: Possibly material for `ferveo` issue

**Review by piotr-roslaniec — APPROVED** [review]

**Review by manumonti — APPROVED** [review]
LGTM! ✌️

File: nucypher/characters/chaotic.py [hunk 2]
```diff
@@ -174,55 +167,54 @@ def gather_encrypted_decryption_shares(
...
+
+        # We can obtain the transcripts from the side-channel (deserialize) and aggregate them
+        validator_messages = [
+            ferveo.ValidatorMessage(validator, transcript)
+            for validator, transcript in self._learner._dkg_insight.transcripts
+        ]
+        aggregate = ferveo.AggregatedTranscript(validator_messages)
+        assert aggregate.verify(
+            self._learner._dkg_insight.shares_num,
+            # TODO this list should have to be passed again (either make `verify` static or use list
```
Comment (derekpierre) [inline nucypher/characters/chaotic.py:185]:
```suggestion
            # TODO this list should not have to be passed again (either make `verify` static or use list
```

File: nucypher/characters/chaotic.py (hunk 1 above)
Comment (derekpierre) [reply nucypher/characters/chaotic.py]: Indeed. This list is passed in the constructor, and then again in the `verify` function - which is redundant.

**Review by KPrasch — CHANGES_REQUESTED** [review]
LGTM - just need to fix the linting errors, and merge the dependency!

**Review by KPrasch — APPROVED** [review]

**Comment by KPrasch** [comment]
I fixed the linting issues in PR #3182

**Comment by derekpierre** [comment]
> I fixed the linting issues in PR #3182

Thanks 🙏

**Pull request merged**